cpu/stm32l4/include/cpu_conf.h

```diff
@@ -36,7 +36,7 @@ extern "C" {
  * @{
  */
 #define CPU_DEFAULT_IRQ_PRIO            (1U)
-#define CPU_IRQ_NUMOF                   (82U)
+#define CPU_IRQ_NUMOF                   (81U)
```

Comment:
Hm, the `stm32l476` has definitely only 81, so the CRS must be specific to the nucleo-l432?! I'd say we go with 81 for this PR and you just add an ifdef case for the `l432` when you add that CPU. Sounds good?